deploy/e2e/service_account.yaml

```diff
@@ -2,3 +2,4 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: e2e-test
+  namespace: default
```

Comment:
can we add a note here saying why we are still using the default ns for tests (TLS fixtures assume default ns)